app/components/landing_page_link_group_component/landing_page_link_group_component.html.slim

```diff
@@ -0,0 +1,5 @@
+div class="#{classes.join(" ")}"
```

Comment:
i think you can use this (to be consistent with other components) `= tag.div(class: classes, **html_attributes) do` and then you can add the shared example tests